homeassistant/components/starlink/coordinator.py

```diff
@@ -55,21 +55,23 @@ def __init__(self, hass: HomeAssistant, name: str, url: str) -> None:
             update_interval=timedelta(seconds=5),
         )
 
+    def _get_starlink_data(self, channel_context: ChannelContext) -> StarlinkData:
+        """Retrieve Starlink data."""
```

Comment:
`channel_context` is also available as an attribute so we don't really need to pass it as a parameter.

```suggestion
    def _get_starlink_data(self) -> StarlinkData:
        """Retrieve Starlink data."""
        channel_context = self.channel_context
```